derex/runner/settings/derex/mongo.py

```diff
@@ -0,0 +1,22 @@
+from xmodule.modulestore.modulestore_settings import update_module_store_settings
+
+
+MONGODB_HOST = "mongodb"
+MONGODB_DB = os.environ.get("MONGO_DB", "mongoedx")
+CONTENTSTORE = {
+    "ENGINE": "xmodule.contentstore.mongo.MongoContentStore",
+    "DOC_STORE_CONFIG": {"host": MONGODB_HOST, "db": MONGODB_DB},
+}
+DOC_STORE_CONFIG = {"host": MONGODB_HOST, "db": MONGODB_DB}
+update_module_store_settings(MODULESTORE, doc_store_settings=DOC_STORE_CONFIG)
+
+# This is needed for the Sysadmin dashboard "Git Logs" tab
+MONGODB_LOG = {"host": DOC_STORE_CONFIG["host"], "db": "xlog"}
+
+if DEREX_PROJECT:
```

Comment:
What is the expected case where `DEREX_PROJECT` is not defined?

If there is none I think we should refrain from suggesting there is one in our code.